src/components/shared/DetailsWidget/DetailsWidget.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { nanoid } from 'nanoid';
+
+import { Colony, ColonyActions, ColonyMotions, User } from '~types';
+import { EventValues } from '~utils/colonyActions';
+
+import DetailItem from './DetailItem';
+import getDetailItems from './detailsWidgetConfig';
+
+const displayName = 'DetailsWidget';
+
+interface Props {
+  actionType: ColonyActions | ColonyMotions;
```

Comment:
I think we should get a bit more consistent with these. In `ActionTypeDetail` component, this type is `EventValues['actionType']`. In #239 you introduced `ActionItemType` which I really like (especially since I then need to change all `ColonyActions` into GraphQL `ColonyActionType` 😅), so could you please make sure you apply it to all these components?